infra/azure/terraform/README.md

```diff
@@ -0,0 +1,21 @@
+# infra/azure
+Terraform scripts and documentation for capz infrastructure that the Kubernetes community runs on Azure.
+The capz folder structure is formatted shown below:
+main.tf
+├── resource_group
+├── identities
+├── key_vault
+├── container_registry
+├── storage_account
+├── role_assignments
+├── variables
+
+# Prerequiste 
+The az-cli-prow service principal is required to be created by a tenant admin. Also to add a federated credentials for the prow ask a tenant admin or the SP admin. iam-config\param.json can be used for creating the fededrated credentials. 

```

Comment:
```suggestion
The `az-cli-prow` service principal with federated credentials is required to be created by a tenant admin. The [iam-config\param.json](./iam-config/param.json) can be used for creating the fededrated credentials. 

```

Reply:
can we also share the example commands to run:

```
az ad sp create-for-rbac --name az-cli-prow
objectid=$(az ad sp list --filter "displayName eq 'az-cli-prow'" --query [0].id --output tsv)
az ad app federated-credential create --id $objectid --parameters param.json

Reply:
s/fededrated/federated